<?xml version="1.0"?>
<launch>
  <rosparam command="load" file="$(find camera_driver)/params/camera_driver.yaml"/>

  <node pkg="camera_driver" type="camera_driver" name="camera_driver" output="screen"/>
</launch>
